bazel/repositories.bzl

```diff
@@ -73,6 +73,10 @@ def _build_recipe_repository_impl(ctxt):
         )
     ctxt.symlink(Label("//ci/prebuilt:BUILD"), "BUILD")
 
+    # Data files for build recipes
+    for f in ["gperftools.patch.gz"]:
```

Comment:
Wouldn't it be easier to update autoconf, or simply hardcode `-DHAVE_MMAP` to ignore the breaking test?

Reply:
@lizan expressed a preference for this approach; for release versions we have fixed autoconf outputs, we will revert back to this when we next move to a release version.